### PR TITLE
RTP12b

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by tcard** [comment]
PTAL

File: Source/ARTRealtimePresence.m [hunk 1]
```diff
@@ -74,7 +75,16 @@ - (void)history:(void (^)(__GENERIC(ARTPaginatedResult, ARTPresenceMessage *) *,
 }
 
 - (BOOL)history:(ARTRealtimeHistoryQuery *)query callback:(void (^)(__GENERIC(ARTPaginatedResult, ARTPresenceMessage *) *, ARTErrorInfo *))callback error:(NSError **)errorPtr {
-    return [super history:query callback:callback error:errorPtr];
+    query.realtimeChannel = self.channel;
+    @try {
+        return [super history:query callback:callback error:errorPtr];
```
Comment (ricardopereira) [inline Source/ARTRealtimePresence.m]: Why did you added a try catch? Is it necessary?

Comment (tcard) [inline Source/ARTRealtimePresence.m]: Yes, [this](https://github.com/ably/ably-ios/blob/master/Source/ARTDataQuery.m#L58) could happen.

Comment (ricardopereira) [inline Source/ARTRealtimePresence.m]: Thanks for clarifying :+1:

**Pull request merged**